source/install-guide/configuring-database.rst

```diff
@@ -4,32 +4,105 @@
 Configuring the database
 ########################
 
-This section describes how to configure the database connection, if required.
+Introduction
+------------
+
+GeoNetwork uses a database to persist aspects such as metadata records, privileges and configurations.
+The database default structure is created by the application on initial startup. Subsequent releases of GeoNetwork 
+will update the database structure automatically. For this reason the database user initially needs create privileges on the database. 
+A number of database dialects are supported; :command:`PostGreSQL`, :command:`Oracle`, :command:`SQL Server`, :command:`H2`.
+This section describes various options to configure the database connection.
+
+H2 database
+-----------
 
 By default, an `H2 <http://www.h2database.com/html/main.html>`_ database is configured
 and created when the application first starts. The H2 database named ``gn.h2.db``
 is created:
 
-* In the ``jetty`` folder of the GeoNetwork application folder when using the :ref:`ZIP distribution <installing-from-zip>`;
+* In the ``jetty`` folder of the GeoNetwork application folder when using the :ref:`ZIP distribution <installing-from-zip>`.
```

Comment:
May wish to use file directive here if it is a folder name.
```
* In the :file:`jetty` folder of the GeoNetwork application folder when using the :ref:`ZIP distribution <installing-from-zip>`.
* In the :file:`bin` folder of Tomcat when deploying the :ref:`WAR <installing-from-war-file>` on Tomcat (started using :file:`startup.sh`).
```

Technically tomcat does not always use startup.sh, it can also run as a windows service ...